pytensor/tensor/rewriting/subtensor.py

```diff
@@ -1751,9 +1774,14 @@ def ravel_multidimensional_bool_idx(fgraph, node):
     x[eye(3, dtype=bool)].set(y) -> x.ravel()[eye(3).ravel()].set(y).reshape(x.shape)
     """
     if isinstance(node.op, AdvancedSubtensor):
-        x, *idxs = node.inputs
+        x = node.inputs[0]
+        tensor_inputs = node.inputs[1:]
     else:
-        x, y, *idxs = node.inputs
+        x, y = node.inputs[0], node.inputs[1]
+        tensor_inputs = node.inputs[2:]
```

Comment:
I don't like the name `tensor_inputs`, x, y are also tensor and inputs. Use `index_variables`?

Reply:
Still using `tensor_inputs` in the last code you pushed